cernopendata/modules/fixtures/data/records/cms-tools-poet.json

```diff
@@ -0,0 +1,101 @@
+    },
+    "accelerator": "CERN-LHC",
+    "authors": [
+      {
+        "name": "CMS Open data group"
+      }
+    ],
+    "collections": [
+      "CMS-Tools"
+    ],
+    "date_created": [
+      "2012"
+    ],
+    "date_published": "2022",
+    "experiment": "CMS",
+    "license": {
+      "attribution": "GNU General Public License (GPL) version 3"
+    },
+    "publisher": "CERN Open Data Portal",
+    "recid": "12501",
+    "run_period": [
+      "Run2012A",
+      "Run2012B",
+      "Run2012C",
+      "Run2012D"
+    ],
+    "source_code_repository": {
+      "url": "https://github.com/cms-opendata-analyses/PhysObjectExtractorTool/tree/2012"
+    },
+    "system_details": {
+      "description": "Use this software in the CMS Open Data VM environment (<a href=\"/record/252\">CMS VM Image for 2011, 2012 and 2015 CMS open data</a>) or in the corresponding <a href=\"/docs/cms-guide-docker\">Docker container</a>.",
+      "release": "CMSSW_5_3_32"
+    },
+    "title": "Physics object extractor tool for the CMS 2012 data",
+    "type": {
+      "primary": "Software",
+      "secondary": [
+        "Tool",
+        "Workflow"
+      ]
+    },
+    "usage": {
+      "description": "<p>POET is an analyzer package to be used with the <a href=\"https://github.com/cms-sw/cmssw\">CMSSW software</a>. CMSSW is available in the CMS Open Data environments: in the CMS VM image and in the CMS Docker container:<ul><li>For the VM image, follow the instructions in step 1 at <a href=\"/VM/CMS/2011\">How to install a CERN Virtual Machine</a>.</li><li> For the Docker container, install the corresponding <a href=\"/docs/cms-guide-docker\">Docker container</a>.</li></ul></p>\n\t\t\t<p>To download and run the POET code, follow the instructions at <a href=\"https://github.com/cms-opendata-analyses/PhysObjectExtractorTool/tree/2012/PhysObjectExtractor\">https://github.com/cms-opendata-analyses/PhysObjectExtractorTool/tree/2012/PhysObjectExtractor</a>.</p>\n<p>For more information about physics object access, see the <a href=\"https://cms-opendata-guide.web.cern.ch/analysis/selection/objects/objects/\">CMS Open Data guide</a>.</p>\n<p>The output files from the POET analyzer package are produced in the <a href=\"https://root.cern.ch/\">ROOT</a> format and their analysis does not require the use of CMSSW and CMS Open data environments anymore.</p>"
```

Comment:
Perhaps use a phrase like "For more information about physics object access, see the[ CMS Open Data Guide] web site" to better indicate that this is something external, and not some guide included on the CERN Open Data portal.

Reply:
Yes, that's good. But keeping the link to the objects page.